scylla-cdc/src/stream_reader.rs

```diff
@@ -43,8 +45,8 @@ trait StreamSession: Sync + Send {
         ids: &[StreamID],
         window_begin: &value::CqlTimestamp,
         window_end: &value::CqlTimestamp,
-        paging_state: Option<Bytes>,
-    ) -> Result<QueryResult, QueryError>;
+        paging_state: PagingState,
+    ) -> Result<(QueryResult, Option<PagingState>), QueryError>;
```

Comment:
:wrench: I suggest returning `PagingStateResponse` here rather than `Option<PagingState>`, to be more in line with the abstract meaning of both.